security/advancedtls/advancedtls.go

```diff
@@ -166,7 +166,7 @@ type ServerOptions struct {
 	// invoke this function every time asked to present certificates to the
 	// client when a new connection is established. This is known as peer
 	// certificate reloading.
-	GetCertificate func(*tls.ClientHelloInfo) (*tls.Certificate, error)
+	GetCertificate func(*tls.ClientHelloInfo) ([]*tls.Certificate, error)
```

Comment:
Should this be `GetCertificates` instead, now that multiple certs can be returned?

Reply:
Yes! Good catch :)